mteb/tasks/Clustering/dan/WikiClusteringP2P.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import annotations
+
+from mteb.abstasks.AbsTaskClustering import AbsTaskClustering
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class WikiClusteringP2P(AbsTaskClustering):
+    metadata = TaskMetadata(
+        name="WikiClusteringP2P",
+        description="Clustering of wikipedia articles inspired by BlubrbsClusteringP2P. Labels are taken from top-level categories as defined in https://da.wikipedia.org/wiki/Kategori:Topniveau_for_emner.",
+        reference=None,
+        dataset={
+            "path": "ryzzlestrizzle/dawiki-clustering-p2p",
+            "revision": "4c4c6b6ad63311d6a072734d86dca1c0b48c441c",
+        },
+        type="Clustering",
+        category="p2p",
+        eval_splits=["test"],
+        eval_langs=["dan-Latn"],
+        main_score="v_measure",
+        date=None,
+        form=None,
+        domains=None,
+        task_subtypes=None,
+        license=None,
+        socioeconomic_status=None,
+        annotations_creators=None,
+        dialect=None,
+        text_creation=None,
+        bibtex_citation=None,
```

Comment:
These need to be filled it, the TaskMetadata object contains descriptions of each field.

Reply:
In the one I copied, there doesn't seem to be any of these - are there any pointers on how to fill them in for this specific dataset?

Reply:
Check out the [adding a dataset](https://github.com/embeddings-benchmark/mteb/blob/main/docs/adding_a_dataset.md) guide or the docstring of the taskmetadata object. Let me know if any of the fields are unclear.

Reply:
Perfect, thank you! I'll fill in all that make sense:))